uk.co.compucorp.civicrm.hrleaveandabsences/js/test/shared/components/leave-request-actions.component.spec.js

```diff
@@ -752,6 +765,10 @@ define([
         leaveRequest.status_id = statuses[sharedSettings.statusNames[status]].value;
       }
 
+      leaveRequest.contact_id = isOwn
+        ? currentlyLoggedInContactId
+        : (parseInt(currentlyLoggedInContactId) + 1).toString();
```

Comment:
why not use `_.uniqueId()` here?

Reply:
Still is a probability to hit the `currentlyLoggedInContactId`. The code above guarantees that it will be different no matter what `currentlyLoggedInContactId` is.